README.md

```diff
@@ -61,19 +64,21 @@ $logger->pushHandler(new StdoutHandler());
 $config = \Kafka\ProducerConfig::getInstance();
 $config->setMetadataRefreshIntervalMs(10000);
 $config->setMetadataBrokerList('10.13.4.159:9192');
-$config->setBrokerVersion('0.9.0.1');
+$config->setBrokerVersion('1.0.0');
 $config->setRequiredAck(1);
 $config->setIsAsyn(false);
 $config->setProduceInterval(500);
-$producer = new \Kafka\Producer(function() {
-	return array(
-		array(
-			'topic' => 'test',
-			'value' => 'test....message.',
-			'key' => 'testkey',
-			),
-	);
-});
+$producer = new \Kafka\Producer(
```

Comment:
It seems we still have some tabs here...

Reply:
daym